underscore.js

```diff
@@ -1125,6 +1125,8 @@
     }
     // Assume equality for cyclic structures. The algorithm for detecting cyclic
     // structures is adapted from ES 5.1 section 15.12.3, abstract operation `JO`.
+    aStack || (aStack = []);
```

Comment:
yeah, that's why i done it this way in first place, and test/index.html passing made me feel like it's ok.